sat6-perf-monitor.yaml

```diff
@@ -1,55 +1,50 @@
 - name: Enable Satellite PCP
-  gather_facts: no
+  gather_facts: false
   hosts: localhost
   connection: local
-
   pre_tasks:
+    - name: Get package facts
+      ansible.builtin.package_facts:
+        manager: auto
 
-    - name: get the package facts
-      package_facts:
-        manager: "auto"
-    
-    - name: get Satellite version
-      set_fact: 
-        satversion: "{{ansible_facts.packages.satellite[0].version}}"
-
+    - name: Identify Satellite version
+      ansible.builtin.set_fact:
+        satellite_version: "{{ ansible_facts['packages']['satellite'][0]['version'] }}"
   tasks:
-
-    - name: Install PCP packages on Satellite 6.15
-      command: >
-        dnf install --disableplugin=foreman-protector -y \
-        python3-policycoreutils \
-        pcp \
-        pcp-pmda-statsd \
-        pcp-system-tools \
-        pcp-pmda-openmetrics \
-        foreman-pcp
-      when: "'6.15' in satversion"
+    - name: Install common PCP packages on Satellite
+      ansible.builtin.dnf:
+        name:
+          - python3-policycoreutils
+          - pcp
+          - pcp-pmda-statsd
+          - pcp-system-tools
+          - pcp-pmda-openmetrics
+        state: present
+        disable_plugin: foreman-protector
+
+    - name: Install additional pcp package on Satellite 6.15
+      ansible.builtin.dnf:
+        name: foreman-pcp
+        state: present
+        disable_plugin: foreman-protector
+      when: satellite_version is version('6.15', '==')
 
     - name: Create hotproc.conf link on Satellite 6.15
-      file:
+      ansible.builtin.file:
         src: /etc/pcp/proc/foreman-hotproc.conf
         dest: /var/lib/pcp/pmdas/proc/hotproc.conf
         state: link
-      when: "'6.15' in satversion"
-
-    - name: Install PCP packages
-      command: >
-        dnf install --disableplugin=foreman-protector -y \
-        python3-policycoreutils \
-        pcp \
-        pcp-pmda-statsd \
-        pcp-system-tools \
-        pcp-pmda-openmetrics
-      when: "'6.15' not in satversion"
-
-    - name: Set pcp_pmcd_t permissive domain
-      selinux_permissive: # we should add UDP port 8125 to statsd_port_t or something similar, but I failed to identify what exactly..
-        name: pcp_pmcd_t
-        permissive: true
+      when: satellite_version is version('6.15', '==')
```

Comment:
Ditto. Previous `'6.15' in satversion` did formally correct check (though not following current standards).